frontend/src/components/Common/ReportForThreeColumns/index.tsx

```diff
@@ -22,27 +28,27 @@ export const ReportForThreeColumns = ({ title, fieldName, listName, data }: Repo
       const [prefix, suffix] = row.name.split('/')
       return (
         <EmojiWrap>
-          <Typography>{prefix}/</Typography>
+          <div>{prefix}/</div>
```

Comment:
Why change it to `<div>`, I think we should keep it, coc follow the constrain of the MetarialUI

Reply:
In fact, it is to prevent the style interference of Typography. Okay,  I'll keep this component.